demos/next-auth/pages/_app.tsx

```diff
@@ -1,10 +1,9 @@
 import { SessionProvider } from "next-auth/react"
-import type { AppProps } from "next/app"
 import "./styles.css"
 
 // Use of the <SessionProvider> is mandatory to allow components that call
 // `useSession()` anywhere in your application to access the `session` object.
-export default function App({ Component, pageProps }: AppProps) {
```

Comment:
Big sads the props are gone, any reason?

Reply:
Something weird with Next types that are not important to fix today!